blueman/plugins/manager/Info.py

```diff
@@ -117,5 +119,8 @@ class Info(ManagerPlugin, MenuItemsProvider):
     def on_request_menu_items(self, manager_menu: ManagerDeviceMenu, device: Device) -> List[Tuple[Gtk.MenuItem, int]]:
         item = create_menuitem(_("_Info"), "dialog-information")
         item.props.tooltip_text = _("Show device information")
-        item.connect('activate', lambda x: show_info(device, manager_menu.get_toplevel()))
+        _window = manager_menu.get_toplevel()
```

Comment:
Similar to earlier, why is the new variable needed?

```python
window = manager_menu.get_toplevel()
assert isinstance(window, Gtk.Window)
```

Reply:
`_window` first gets implicitly defined as optional, the assertion narrows it down to non-optional, but if we use it in the lambda it will be considered optional again (https://github.com/python/mypy/issues/2608). Opposed to that, `window` gets defined as the narrowed type of `_window` so it is non-optional right away.